app/src/main/res/layout/location_fragment_fragment.xml

```diff
@@ -10,32 +10,45 @@
         android:layout_height="match_parent"
         android:visibility="visible"
         tools:visibility="visible" />
+
     <LinearLayout
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:orientation="horizontal"
-        android:padding="10dp">
+        android:id="@+id/topMapContainer"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center_horizontal"
+        android:orientation="vertical">
 
-        <Button
-            android:id="@+id/SGWBtn"
+        <LinearLayout
+            android:id="@+id/campusSwitchBtnContainer"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginRight="5dp"
-            android:layout_weight="1"
-            android:background="@color/colorAccent"
-            android:text="SGW"
-            android:textColor="@color/textColor"></Button>
+            android:layout_marginStart="1dp"
+            android:layout_marginEnd="1dp"
+            android:foregroundGravity="center"
+            android:orientation="horizontal"
+            android:padding="10dp">
 
-        <Button
-            android:id="@+id/loyolaBtn"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginLeft="5dp"
-            android:layout_weight="1"
-            android:background="@color/colorAccent"
-            android:text="Loyola"
-            android:textColor="@color/textColor"></Button>
+            <Button
+                android:id="@+id/SGWBtn"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginRight="5dp"
+                android:layout_weight="1"
+                android:background="@color/colorAccent"
```

Comment:
Didn't the PO also request for the buttons to look less "disabled"? I'd suggest changing these button colours to @color/colorAppTheme.

Reply:
i would do this in this pr if possible

Reply:
<img width="435" alt="Screen Shot 2020-03-06 at 4 54 14 PM" src="https://user-images.githubusercontent.com/34817695/76125599-24493a00-5fcb-11ea-96d9-02c0b4860467.png">
I don't get it, how do they look disabled? what color should I put

Reply:
They are the same color in info card, so to stay consistent, I thought we keep them like this

Reply:
<img width="261" alt="Screen Shot 2020-03-06 at 5 19 10 PM" src="https://user-images.githubusercontent.com/34817695/76127009-9e2ef280-5fce-11ea-8553-d0d1f1053ea0.png">
is this good?

Reply:
@thomasgauvin @viveanban